tests/graphql_directives/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -45,10 +45,10 @@ fragment YouCanHaveDirectivesHereToo on SomeType @yesReally(what: "yes") {
   }
 
   thisFieldHasALotOfDirectives
-    @thisIsthefirst
-    @thisIsTheSecond
-    @thisIsTheThird
-    @thisIstheFourthWillBeTooLongForSure(and: "it has arguments as well")
+  @thisIsthefirst
```

Comment:
IMO this is a regression - it's harder to read what it applies to when it's not indented.

e.g.

```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String! @deprecated @isAuthenticated
  versions: Versions!
}
```

Prettier 2.0.5:
```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String!
    @deprecated
    @isAuthenticated
  versions: Versions!
}
```

Prettier 2.1.1:
```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String!
  @deprecated
  @isAuthenticated
  versions: Versions!
}
```

Especially since they are applied to the thing before rather than. after it, unlike decorators in JS or annotations in Java

Reply:
@SimenB Let's open an issue, agree

Reply:
can do

Reply:
#9068